### PR TITLE
Added html and regex to reduce false positives

### PR DESCRIPTION
**Pull request by aryangupta701**
- If the content is of HTML type then skip 
- Removes String.contains() and uses regex

### CODE REVIEW

**Comment by aryangupta701** [comment]
@itsdivyanshjain  Please review

File: src/main/zapHomeFiles/json/hidden_files_astra.json
Comment (itsdivyanshjain) [inline src/main/zapHomeFiles/json/hidden_files_astra.json:1]: can we also improve checks for one word base check like in readme files & more..

**Comment by itsdivyanshjain** [comment]
Apart from one comment all things look goods to me

**Comment by aryangupta701** [comment]
@itsdivyanshjain Please review again I have made the necessary changes.

**Comment by itsdivyanshjain** [comment]
approved
cc: @killshotrevival

**Review by killshotrevival — APPROVED** [review]

**Comment by killshotrevival** [comment]
Good wok @aryangupta701 😄

**Pull request merged**